publication-commons/src/main/java/no/unit/nva/publication/RequestUtil.java

```diff
@@ -72,4 +81,33 @@ public static String getOwner(RequestInfo requestInfo) throws ApiGatewayExceptio
         }
         throw new InputException(MISSING_CLAIM_IN_REQUEST_CONTEXT + CUSTOM_FEIDE_ID, null);
     }
+
+    /**
+     * Get pagesize from request query parameters.
+     *
+     * @param requestInfo requestInfo
+     * @return the pagesize if given, otherwise DEFAULT_PAGESIZE
+     * @throws ApiGatewayException exception thrown if value is not legal positive integer
+     */
+    public static int getPageSize(RequestInfo requestInfo) throws ApiGatewayException {
```

Comment:
Bruk RequestUtils.getQueryParameter fra nva-commons. Denne klassen blir da overflødig.

Reply:
This is left as is because of handling of optional parameter pagesize and converting to int, using Requestutils result in same amount of nesting :-(